_sass/elements/_hex-icon.scss

```diff
@@ -0,0 +1,43 @@
+:root {
+  --sin-60: 0.866;
+}
+
+.hex-icon {
+  --icon-height: 48px;
+
+  width: calc(var(--icon-height) * var(--sin-60));
+  height: var(--icon-height);
+  display: inline-block;
+  margin-right: 12px;
+  position: relative;
+
+  filter: drop-shadow(0 0 2px rgba(0,0,0,0.1))
+    drop-shadow(0 1px 1px rgba(0,0,0,0.2))
+    drop-shadow(0 1px 1px rgba(0,0,0,0.1));
+
+  .inner {
+    background-color: white;
+    width: 100%;
+    height: 100%;
+
+    clip-path: polygon(
+      evenodd,
+      0% 25%,
+      0% 75%,
+      50% 100%,
+      100% 75%,
+      100% 25%,
+      50% 0,
+      0% 25%
+    );
+
+    & > * {
+      position: absolute;
+      width: 24px;
+      height: 24px;
```

Comment:
**nitpick:** Looks like these values could be expressed as functions of `--icon-height`.
```suggestion
      width: calc(var(--icon-height) / 2);
      height: calc(var(--icon-height) / 2);
```